tests/include/mock_custom_nrf_modem_at.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef MOCK_CUSTOM_NRF_MODEM_AT_H_
+#define MOCK_CUSTOM_NRF_MODEM_AT_H_
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @file mock_custom_nrf_modem_at.h
+ * @brief Public APIs for the Custom nrf_modem_at mock.
+ * @defgroup mock_custom_nrf_modem_at Custom mock for nrf_modem_at
+ * @{
+ */
+
+/* TODO: Add __wrap_nrf_modem_at_scanf_Ignore */
```

Comment:
I've removed this TODO.
I started to implement ignore and couldn't really find a place to test it because ignoring `nrf_modem_at_scanf()` call impacts behavior of the caller which doesn't get meaningful response and ends up going into some error branches easily.
I can implement ignoring to the mock when there is a use case that benefits from it.